spotbugs/etc/messages_ja.xml

```diff
@@ -5492,48 +5552,46 @@ static フィールドにすることを検討してください。
 
   <BugPattern type="UC_USELESS_VOID_METHOD">
     <ShortDescription>役に立たない空ではない void メソッド</ShortDescription>
-    <LongDescription>メソッド {1} は、役に立たないようです。</LongDescription>
+    <LongDescription>メソッド {1} は，役に立たないようです。</LongDescription>
     <Details>
 <![CDATA[
 <p>
-我々の解析は、この空ではない void メソッドが実際に有用な仕事を行わないことを示しています。
-確認してください。おそらくそのコードが間違っているか、またはボディを完全に除去できます。
+我々の解析は，この空ではない <code>void</code> メソッドが実際に有用な仕事を行わないことを示しています。
+確認してください。おそらくそのコードが間違っているか，またはボディを完全に除去できます。
 </p>
 <p>
-我々はできる限り誤検出を減らそうと努力しているが、いくつかのケースでは警告は間違っているかもしれません。
+我々はできる限り誤検出を減らそうと努力しているが，いくつかのケースでは警告は間違っているかもしれません。
 よくある誤検出例です。
 </p>
-<p>
-- メソッドは、副作用を持つかもしれないクラスのロードをトリガすることを意図している
-</p>
-<p>
-- メソッドは、暗黙のわかりにくい例外をスローするように意図されている
-</p>
+<ul>
+  <li>メソッドは，副作用を持つかもしれないクラスのロードをトリガーすることを意図している</li>
+  <li>メソッドは，暗黙のわかりにくい例外をスローするように意図されている</li>
+</ul>

```

Comment:
Improve HTML formatting.
- Modify source code block(&lt;blockquote&gt; -> &lt;code&gt;).
- Change to the bulleted list(&lt;ul&gt;&lt;/ul&gt;).
- Change Datails of the BugCategory into the HTML format.

Is there anything else you need to improve?